compiler/codegen/Relocation.cpp

```diff
@@ -423,6 +423,43 @@ const char *TR::ExternalRelocation::_externalRelocationTargetKindNames[TR_NumExt
    "TR_DebugCounter (59)",
    "TR_ClassUnloadAssumption (60)",
    "TR_J2IVirtualThunkPointer (61)",
+   "TR_InlinedAbstractMethodWithNopGuard (62)",
```

Comment:
This one doesn't look like it should fall into the category of "Validation type"

Reply:
You're right, it isn't; it's just at the bottom because moving it to where the other inlined methods are would involve shifting and renaming (the number portion) or all of the other relocation names, which in the interest of time I decided was not worth doing.

Reply:
Yeah I did, because there wasn't really much to be gained by logically separating the commits that effectively did the same thing, namely add new relo kinds.